build/RepositoryBuild.targets

```diff
@@ -56,7 +57,8 @@
       StopOnFirstFailure="false"
       Targets="_TestRepository"
       Properties="BuildNumber=$(BuildNumber);IsFinalBuild=$(IsFinalBuild);Configuration=$(Configuration);_NoBuildRepos=$(_NoBuildRepos)"
-      ContinueOnError="true">
+      ContinueOnError="true"
+      Condition=" @(_BatchedTestRepo.SkipTests) != 'true' ">
```

Comment:
@natemcmaster This doesn't seem to do anything

Reply:
```suggestion
      Condition=" '%(_BatchedTestRepo.SkipTests)' != 'true' ">
```

Might be due to wrong syntax...

Reply:
Oh, the quotes...